packages/homepage/src/screens/home/video/index.js

```diff
@@ -120,7 +120,8 @@ const Video = () => {
       animate={{ opacity: 1, y: 0 }}
       transition={{ delay: 1, duration: 1, ease: 'easeOut' }}
       css={`
-        overflow: active ? "hidden" : 'initial';
+        overflow-x: ${active ? 'initial' : 'hidden'};
```

Comment:
Nice catch with the syntax error 👍 

Perhaps we should make this always `hidden`, since there is still some horizontal scroll when the animation is transitioning. What do you think?

Reply:
Sounds reasonable. Thank you for pointing it out, I updated the PR.